docfx_yaml/extension.py

```diff
@@ -1002,6 +1031,43 @@ def pretty_package_name(package_group):
     return " ".join(capitalized_name)
 
 
+# Given generated markdown files, incorporate them into the docfx_yaml output.
+# The markdown file metadata will be added to top level of the TOC.
+def find_markdown_pages(app, outdir):
+    # Use this to ignore markdown files that are unnecessary.
+    files_to_ignore = [
+        "index.md",     # merge index.md and README.md and index.yaml later
+        "reference.md", # Reference docs overlap with Overview. Will try and incorporate this in later.
```

Comment:
File an issue and reference it here?

Reply:
Do you mean within the code, or in this PR? If it's the latter, done.

Reply:
Both.